api/base_controller.go

```diff
@@ -218,13 +218,19 @@ func (c *BaseController) PatchObject(r *web.Request) (*web.Response, error) {
 	if err := util.BytesToObject(r.Body, objFromDB); err != nil {
 		return nil, err
 	}
+
 	objFromDB.SetID(objectID)
 	objFromDB.SetCreatedAt(createdAt)
 	objFromDB.SetUpdatedAt(updatedAt)
+
+	labels, _, _ := query.ApplyLabelChangesToLabels(labelChanges, objFromDB.GetLabels())
+	objFromDB.SetLabels(labels)
```

Comment:
it seems this is done twice - here and in `Update`, why?

Reply:
To handle the case when intrrceptors add label changes